[2024-06-05T21:52:57.483+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T21:52:57.508+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestion_data.insertar_datos manual__2024-06-05T21:49:46.721221+00:00 [queued]>
[2024-06-05T21:52:57.518+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestion_data.insertar_datos manual__2024-06-05T21:49:46.721221+00:00 [queued]>
[2024-06-05T21:52:57.519+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-05T21:52:57.530+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insertar_datos> on 2024-06-05 21:49:46.721221+00:00
[2024-06-05T21:52:57.538+0000] {standard_task_runner.py:63} INFO - Started process 1294 to run task
[2024-06-05T21:52:57.542+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingestion_data', 'insertar_datos', 'manual__2024-06-05T21:49:46.721221+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/dag_entregable_fedebar.py', '--cfg-path', '/tmp/tmp0qepomo5']
[2024-06-05T21:52:57.544+0000] {standard_task_runner.py:91} INFO - Job 136: Subtask insertar_datos
[2024-06-05T21:52:57.604+0000] {task_command.py:426} INFO - Running <TaskInstance: ingestion_data.insertar_datos manual__2024-06-05T21:49:46.721221+00:00 [running]> on host 167cb9f6b96a
[2024-06-05T21:52:57.696+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='FedeBar' AIRFLOW_CTX_DAG_ID='ingestion_data' AIRFLOW_CTX_TASK_ID='insertar_datos' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T21:49:46.721221+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-05T21:49:46.721221+00:00'
[2024-06-05T21:52:57.698+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T21:52:57.709+0000] {base.py:84} INFO - Using connection ID 'coder-redshift-fede' for task execution.
[2024-06-05T21:52:57.710+0000] {weather_info_ETL.py:23} INFO - Conectando a Redshift en data-engineer-cluster.cyhh5bfevlmn.us-east-1.redshift.amazonaws.com:5439/data-engineer-database
[2024-06-05T21:52:58.873+0000] {weather_info_ETL.py:32} INFO - Conectado a Redshift con éxito!
[2024-06-05T21:52:58.879+0000] {warnings.py:109} WARNING - /opt/***/dags/weather_info_ETL.py:184: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  existing_data = pd.read_sql(existing_data_query, conn)

[2024-06-05T21:52:59.492+0000] {warnings.py:109} WARNING - /opt/***/dags/weather_info_ETL.py:187: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  max_existing_load_date = pd.read_sql(max_existing_load_date_query, conn).iloc[0, 0]

[2024-06-05T21:52:59.692+0000] {weather_info_ETL.py:170} ERROR - Archivo no encontrado: [Errno 2] No such file or directory: 'C:\\Fede\\Data Engineer Coderhouse\\Coder-entregable-3\\***_docker\\poblacion.csv'
[2024-06-05T21:53:00.016+0000] {weather_info_ETL.py:214} ERROR - Error al insertar datos en Redshift: INSERT has more target columns than expressions

[2024-06-05T21:53:00.017+0000] {weather_info_ETL.py:218} INFO - Conexión a Redshift cerrada.
[2024-06-05T21:53:00.018+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T21:53:00.019+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T21:53:00.031+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ingestion_data, task_id=insertar_datos, run_id=manual__2024-06-05T21:49:46.721221+00:00, execution_date=20240605T214946, start_date=20240605T215257, end_date=20240605T215300
[2024-06-05T21:53:00.093+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T21:53:00.119+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-05T21:53:00.122+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
